doc/xds-test-descriptions.md

```diff
@@ -779,4 +779,50 @@ that during some 10-second interval, all traffic goes to the other four
 backends and all requests end with the `OK` status.
 3.  The test driver removes the client configuration to send metadata. The
 driver asserts that during some 10-second interval, traffic is equally
-distributed among the five backends, and all requests end with the `OK` status.
\ No newline at end of file
+distributed among the five backends, and all requests end with the `OK` status.
+
+### custom_lb
+This test verifies that a custom load balancer policy can be configured in the
+client. It also verifies that when given a list of policies the client can
+ignore a bad one and try the next one on the list until it finds a good one.
+
+Client parameters:
+
+1.  --num_channels=1
+2.  --qps=100
+
+Load balancer configuration:
+
+One MIG with a single backend.
```

Comment:
Please clarify... I have not seen this acronym before so it may confuse other people implementing tests as well.

Reply:
MIG = Managed Instance Group, used to manage multiple VMs as a group
https://cloud.google.com/compute/docs/instance-groups

It's used elsewhere in this file so I just followed the pattern.